saiku-bi-platform-plugin-p5/src/main/plugin/components/saikuWidget/SaikuWidgetComponent.js

```diff
@@ -32,7 +32,7 @@ var saikuWidgetComponent = BaseComponent.extend({
 			htmlId = t;
 		}
 		var myClient = new SaikuClient({
-		    server: "../../../plugin/saiku/api",
+		    server: "/pentaho/plugin/saiku/api",
```

Comment:
i wouldn't do that. just simply do a:
var webappName = window.location.pathname.split("/")[1];
....
server: webappName + "/plugin/saiku/api",